.github/workflows/android.yaml

```diff
@@ -90,7 +90,7 @@ jobs:
             ~/.android/adb*
           key: avd-${{ runner.os }}
       - name: Start AVD then run E2E tests
-        uses: reactivecircus/android-emulator-runner@v2
+        uses: reactivecircus/android-emulator-runner@b530d96654c385303d652368551fb075bc2f0b6b
```

Comment:
verify visually by noticing the `v2` tag is currently attached to this SHA here: https://github.com/ReactiveCircus/android-emulator-runner/commit/b530d96654c385303d652368551fb075bc2f0b6b